src/interfaces/GlobalTypes.ts

```diff
@@ -20,4 +20,4 @@ export interface TokenDto {
 	jti: string;
```

Comment:
What is this? The JWT?

Reply:
This was already committed. Also, I don't see it being used.

From JWT website, it says "JWT ID (Unique identifier for this token)".

```json
{
  "token_type": "access",
  "exp": 1601751234,
  "jti": "33c7889f519349f5b98edec1124e9ca!",
  "user_id": 1
}
```